src/utils/__tests__/rttc.ts

```diff
@@ -6,6 +6,8 @@ const num = 0
 const bool = true
 const str = ' '
 const func = mockClosure()
```

Comment:
we should test both `mockClosure` and builtin functions, as well as `undefined`.

Reply:
Done, and added null as well.